src/renderers/shared/ReactDebugTool.js

```diff
@@ -18,8 +19,65 @@ var ReactChildrenMutationWarningHook = require('ReactChildrenMutationWarningHook
 var ExecutionEnvironment = require('ExecutionEnvironment');
 
 var performanceNow = require('performanceNow');
+var invariant = require('invariant');
 var warning = require('warning');
 
+import type { ReactElement } from 'ReactElementType';
+import type { DebugID } from 'ReactInstanceType';
+
+type Hook = any;
+type Payload = mixed;
+
+type TimerType =
```

Comment:
This will probably need to be moved to a different place later. @cpojer told me during his jest flowification hackathon that it's better to put them inside of the file you're working on and when you need it in another place, then extract them, instead of doing it the other way.